src/collections/_documentation/error-reporting/configuration/before-send/php.md

```diff
@@ -0,0 +1,14 @@
+In PHP a function can be used to modify the event or return a completely new one. If you return `null`, the event will be discarded.
```

Comment:
FWIW this phrasing is already used for the other files in this directory. See the first code sample at https://docs.sentry.io/error-reporting/configuration/filtering/ I would keep it.

>It will be called right before the event is sent to the server, so it's the last place where you can edit its data.

I like this sentence. I think it could come before the toggle as it's not lang specific content.